web/impact/impact/tests/test_create_edit_office_hours_view.py

```diff
@@ -269,6 +344,11 @@ def _assert_update_office_hour(self, office_hour):
             pk=office_hour.id)
         self.assertEqual(updated_office_hour.topics, self.updated_topics)
 
+    def _assert_office_hour_not_updated(self, office_hour):
+        updated_office_hour = MentorProgramOfficeHour.objects.get(
+            pk=office_hour.id)
+        self.assertEqual(updated_office_hour.topics, office_hour.topics)
```

Comment:
What if the test updates start or end time rather than topics?

Reply:
I have updated `_assert_office_hour_not_updated` to check for all the office hours fields